### PR TITLE
nodejs-fr: bootstrap about/governance page

### PR DESCRIPTION
**Pull request by Tiriel**
Hey there!

Another first draft, hopefully with less typos than the first.
I'm open to revisions as usual!

Cheers

### CODE REVIEW

File: locale/fr/about/governance.md
Comment (yoannmoinet) [inline locale/fr/about/governance.md]: Unifier les majuscules dans la liste au début de chaque ligne.

et

> la liste des Collaborateurs

File: locale/fr/about/governance.md
Comment (yoannmoinet) [inline locale/fr/about/governance.md]: > **q**ui

pas besoin de majuscule ici.

File: locale/fr/about/governance.md
Comment (yoannmoinet) [inline locale/fr/about/governance.md]: > se voient accord**er**

File: locale/fr/about/governance.md
Comment (yoannmoinet) [inline locale/fr/about/governance.md]: > un autre Collaborateur

File: locale/fr/about/governance.md
Comment (yoannmoinet) [inline locale/fr/about/governance.md]: > est proposé sur

File: locale/fr/about/governance.md
Comment (yoannmoinet) [inline locale/fr/about/governance.md]: > ou par une motion

File: locale/fr/about/governance.md
Comment (yoannmoinet) [inline locale/fr/about/governance.md]: > la situation

File: locale/fr/about/governance.md
Comment (yoannmoinet) [inline locale/fr/about/governance.md]: > approuvé par le TSC

File: locale/fr/about/governance.md
Comment (yoannmoinet) [inline locale/fr/about/governance.md]: > quel Collaborateur

File: locale/fr/about/governance.md
Comment (yoannmoinet) [inline locale/fr/about/governance.md]: > en ajoutant

File: locale/fr/about/governance.md
Comment (yoannmoinet) [inline locale/fr/about/governance.md]: > un objet

**Review by yoannmoinet — CHANGES_REQUESTED** [review]
Thank you for that!

Not much to change, mostly small typohs.

I still need to check for translation issues. I'll do it later in another review.

**Comment by Tiriel** [comment]
Thanks! Typos fixed, I'll wait for your other review.

By the way, how can one join the website team and/or one of the localization teams? I'm currently investing myself hard, and I volunteered already in multiples issues (here, in the nodejs-fr repo, and in the CommComm repo and sent application to the team)

**Comment by fhemberger** [comment]
@Tiriel Usually, as you contribute, you get added to the Website WG's contributors list (this is a manual process at the moment done every couple of months), thus making you a full website team member.

Somehow you slipped through (maybe because you were already an org member), so I just added you manually. Sorry for that! As for nodejs-fr, @snostorm, can you please add him as well?

**Comment by Tiriel** [comment]
@fhemberger No problem at all, and thanks a lot for adding!
This is awesome! (And that must be like the tenth time I said it on one of the nodejs repos, but I stand by it).

File: locale/fr/about/governance.md
Comment (cedced19) [inline locale/fr/about/governance.md]: au-delà

File: locale/fr/about/governance.md
Comment (cedced19) [inline locale/fr/about/governance.md]: expérience

File: locale/fr/about/governance.md
Comment (cedced19) [inline locale/fr/about/governance.md]: r**e**sponsabilité

File: locale/fr/about/governance.md
Comment (cedced19) [inline locale/fr/about/governance.md]: changement

File: locale/fr/about/governance.md
Comment (cedced19) [inline locale/fr/about/governance.md]: mod**é**rateur

File: locale/fr/about/governance.md
Comment (cedced19) [inline locale/fr/about/governance.md]: décisionnaire

File: locale/fr/about/governance.md
Comment (cedced19) [inline locale/fr/about/governance.md]: cl**ô**ture

**Review by cedced19 — CHANGES_REQUESTED** [review]

**Comment by Tiriel** [comment]
Fixed ;)

File: locale/fr/about/governance.md
Comment (yoannmoinet) [inline locale/fr/about/governance.md]: > La direction technique

File: locale/fr/about/governance.md
Comment (yoannmoinet) [inline locale/fr/about/governance.md]: > conduite

No need for uppercase I think.

File: locale/fr/about/governance.md
Comment (yoannmoinet) [inline locale/fr/about/governance.md]: > significative avec la gestion du projet.

File: locale/fr/about/governance.md
Comment (yoannmoinet) [inline locale/fr/about/governance.md]: > sont faites Collaborateurs

File: locale/fr/about/governance.md
Comment (yoannmoinet) [inline locale/fr/about/governance.md]: > les droits de commit sur

Or, update the `droits de commit` in the next paragraph by `droits en écriture` to keep the same term for both.

File: locale/fr/about/governance.md
Comment (yoannmoinet) [inline locale/fr/about/governance.md]: > les droits de commit

See previous note.

File: locale/fr/about/governance.md
Comment (yoannmoinet) [inline locale/fr/about/governance.md]: > Les modifications du contenu du dépôt nodejs/node sont faites de

File: locale/fr/about/governance.md
Comment (yoannmoinet) [inline locale/fr/about/governance.md]: > d'assumer l'entière responsabilité du changement

File: locale/fr/about/governance.md
Comment (yoannmoinet) [inline locale/fr/about/governance.md]: > un Collaborateur additionnel est requis

File: locale/fr/about/governance.md
Comment (yoannmoinet) [inline locale/fr/about/governance.md]: > voir le README.md du projet.

File: locale/fr/about/governance.md
Comment (yoannmoinet) [inline locale/fr/about/governance.md]: > voir le README.md du projet.

File: locale/fr/about/governance.md
Comment (yoannmoinet) [inline locale/fr/about/governance.md]: > est maintenu sur

File: locale/fr/about/governance.md
Comment (yoannmoinet) [inline locale/fr/about/governance.md]: The sentence seems odd in comparaison with the source material, here's what I propose : 

> Cependant, l'objectif recherché est entre 6 et 12 personnes, afin d'assurer une couverture adéquate des différents domaines d'expertise, en balance avec une prise de décision efficace.

File: locale/fr/about/governance.md
Comment (yoannmoinet) [inline locale/fr/about/governance.md]: In french,  [1/3 is singular and should be written in full letters](http://www.aidenet.eu/grammaire04k.htm), not sure about english.

> Pas plus d'un tiers des membres du TSC ne pourra être affilié au même employeur.

File: locale/fr/about/governance.md
Comment (yoannmoinet) [inline locale/fr/about/governance.md]: > plus d'un tiers des membres du TSC partage

File: locale/fr/about/governance.md
Comment (yoannmoinet) [inline locale/fr/about/governance.md]: Not sure about the `-s` here. I'd use the plural form only, like:

> le ou les employeurs sur représentés.

File: locale/fr/about/governance.md
Comment (yoannmoinet) [inline locale/fr/about/governance.md]: > de manière continue sur GitHub

File: locale/fr/about/governance.md
Comment (yoannmoinet) [inline locale/fr/about/governance.md]: > géré par l'ensemble des Collaborateurs.

File: locale/fr/about/governance.md
Comment (yoannmoinet) [inline locale/fr/about/governance.md]: > Ni le modérateur ni le TSC ne peuvent

File: locale/fr/about/governance.md
Comment (yoannmoinet) [inline locale/fr/about/governance.md]: > de l'envoyer

File: locale/fr/about/governance.md
Comment (yoannmoinet) [inline locale/fr/about/governance.md]: > la majorité du TSC, sinon la discussion continue.

**Review by yoannmoinet — CHANGES_REQUESTED** [review]
Thank you for the update, I've now reviewed the meaning of it all.

Your translation is really good, your terms are really well-found and the meaning stays mostly in par with the source. That's a great work of translation.

I've just added some nitpicks that I found diverged from the source.

**Comment by Tiriel** [comment]
Thanks! I made changes as requested, I just slightly adapted one phrase to make it sound better.

File: locale/fr/about/governance.md
Comment (yoannmoinet) [inline locale/fr/about/governance.md]: missing that first `La` uppercase 🤓

File: locale/fr/about/governance.md
Comment (yoannmoinet) [inline locale/fr/about/governance.md]: I think you missed that last `)`

File: locale/fr/about/governance.md
Comment (yoannmoinet) [inline locale/fr/about/governance.md]: :+1:

File: locale/fr/about/governance.md
Comment (yoannmoinet) [inline locale/fr/about/governance.md]: This sentence sounds weird to me now.

Why removing the `et` part and not applying my suggestion of `par l'ensemble des Collaborateurs`?

It's more in par with the source material I think.

**Review by yoannmoinet — CHANGES_REQUESTED** [review]
Small review.

File: locale/fr/about/governance.md
Comment (Tiriel) [reply locale/fr/about/governance.md]: Oh my. Sorry, I hadn't seen the last part of your suggestion. Correcting ASAP

**Comment by Tiriel** [comment]
Done! I hope I didn't forget anything this time :D

**Review by yoannmoinet — APPROVED** [review]
Looks good to me!

Thank you very much for following through it all !

Just need the approval from @cedced19 now, to ensure his review updates are ok.

File: locale/fr/about/governance.md
Comment (clakech) [inline locale/fr/about/governance.md:11]: 'de haut-niveau' ne me semble pas naturel, peut être le supprimer ?

File: locale/fr/about/governance.md
Comment (clakech) [inline locale/fr/about/governance.md:101]: ajouter un lien ici ?

File: locale/fr/about/governance.md
Comment (clakech) [inline locale/fr/about/governance.md:103]: objets => sujets ?

File: locale/fr/about/governance.md
Comment (clakech) [inline locale/fr/about/governance.md:125]: this link should be nodejs/build nope ?

**Review by clakech — APPROVED** [review]
great thanks !

**Comment by clakech** [comment]
@fhemberger ok for me, can we remove the on hold tag ?

**Comment by fhemberger** [comment]
@clakech Waiting for final feedback from @cedced19, then it can be merged.

File: locale/fr/about/governance.md
Comment (Tiriel) [reply locale/fr/about/governance.md:11]: C'était pour essayer de coller au plus près de l'original. 'gouvernance générale'?

File: locale/fr/about/governance.md
Comment (Tiriel) [reply locale/fr/about/governance.md:101]: Ca peut-être une bonne idée oui. Mais dans ce cas, peut-être voir pour l'ajouter dans toutes les trad et surtout sur le site `en` non?

File: locale/fr/about/governance.md
Comment (Tiriel) [reply locale/fr/about/governance.md:125]: Didn't touch the link. @nodejs/website can someone answer? Do we need to change the link? On all pages?

Comment (lpinca) [reply locale/fr/about/governance.md:125]: It redirects to https://github.com/nodejs/build but yes it should be updated. It is probably better to do it in a dedicated PR though.

**Comment by Tiriel** [comment]
Another ping to @cedced19 just in case, but since all his suggested changes have been adressed, maybe this can be merged tomorrow in any case?

**Comment by fhemberger** [comment]
I'm merging this now. The changes are addressed, details can be added in an additional PR later.

**Pull request merged**
